### PR TITLE
Docker & Postgres

### PR DESCRIPTION
**Pull request by pavlov200912**
- Memory storage has been replaced by postgres database.
- Docker compose file has been created.
- Server and storage are wrapped in docker containers.  
- Small refactorings in link.go (DeleteLink should verify accountId inside the LinkUseCases inteface, not in the memory interface)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,38 @@
...
+Скачайте докер и запустите команду:
+```
+docker-compose up
+```
+
+## Пример запросов:
+
+Регистрация
+```
+requests.post("http://localhost:8080/signup", json={"login": "ivanpavlov", "password": "SomeComplicated2131"})
```
Comment (M0nsteR) [inline README.md:27]: What is this? What is Python code doing here?

**Review by M0nsteR — APPROVED** [review]
Good to go.

File: README.md (hunk 1 above)
Comment (pavlov200912) [reply README.md:27]: That is my way of doing post/get requests. (I can't use curl on windows properly). You might think this is pseudocode.  Seems readable to me.

Comment (M0nsteR) [reply README.md:27]: This is all very well, but please think about the readers who might be misled by this. You'd better mention the language of choice explicitly, like "from Python .. run this" or "using curl .. run that".

Comment (M0nsteR) [reply README.md:27]: And of course you know how to spell "usage"  :)

**Pull request merged**